packages/uhk-web/src/styles/_themes.scss

```diff
@@ -0,0 +1,158 @@
+$dark-theme: false !default;
```

Comment:
Would not be better to use 2 theme file instead of 1? 

If we will use 2 files I see the following benefits
- the `$dark-theme` variable not needed
- the `@if $dark-theme` condition not needed, less indentation
- we can compare light and dark theme file much easier and we can see detect much easier the different things

Reply:
That's a fair point. I was inventing this along as I went but they could definely be separate files at this point.

Reply:
They are now separated in their own themes folder.

Reply:
thx